src/command.ts

```diff
@@ -36,21 +36,33 @@ function takeScreenshot(subject: any, name: string, screenshotOptions: any): voi
     objToOperateOn = cy
   }
 
+  let screenshotPath: string
   // eslint-disable-next-line promise/catch-or-return
-  objToOperateOn.screenshot(name, screenshotOptions).then(() => {
-    return null
-  })
+  objToOperateOn
+    .screenshot(name, {
+      ...screenshotOptions,
+      onAfterScreenshot(_el: any, props: any) {
+        screenshotPath = props.path
+      }
+    })
+    .then(() => {
+      return cy.wrap(screenshotPath).as('screenshotAbsolutePath')
```

Comment:
Now, the first difficulty starts here, where the elegant way of passing this absolute path down to other chainable commands eludes me.
The only thing that works is this cy.wrap + cy.get combo.